.github/workflows/build.yml

```diff
@@ -405,6 +407,8 @@ jobs:
     - name: "[macOS/Windows] Upload build to downloads.mixxx.org"
       # skip deploying Ubuntu builds to downloads.mixxx.org because these are deployed to the PPA
       if: runner.os != 'Linux' && github.event_name == 'push' && env.SSH_AUTH_SOCK != null
+      # FIXME: According to the schema expression syntax ${{ }} is required for the
+      # argument of `shell:`` in contradiction to the official GitHub docs!?
       shell: bash --login -eo pipefail "{0}"
```

Comment:
The schema description seems overly strict.
```JSON
"shell": {
  "$comment": "https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell",
  "description": "You can override the default shell settings in the runner's operating system using the shell keyword. You can use built-in shell keywords, or you can define a custom set of shell options.",
  "type": "string",
  "anyOf": [
    {
      "$ref": "#/definitions/expressionSyntax"
},
    {
      "$comment": "https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#custom-shell",
      "enum": [
        "bash",
        "pwsh",
        "python",
        "sh",
        "cmd",
        "powershell"
      ]
    }
  ]
},
```

Reply:
The `${{ }}` seems to come from the `expressionSyntax` definition.

Reply:
I opened a PR at https://github.com/SchemaStore/schemastore/pull/2076.